demo/lib/App.tsx

```diff
@@ -441,3 +483,25 @@ const SidebarFooter = () => (
     </p>
   </div>
 );
+
+const Alert = ({ visible, positive }) => {
+  if (visible) {
```

Comment:
If not visible doesn't return anything? If so, maybe we can remove the prop and do:

`this.state.message.visible && <Alert positive={this.state.message.positive} />`